### PR TITLE
Upgrades to forward modeling util

### PR DESCRIPTION
**Pull request by jfcrenshaw**
Just a few small upgrades to the forward modeling util that were useful for tests I was running.

### CODE REVIEW

File: python/lsst/ts/wep/utils/modelUtils.py [hunk 1]
```diff
@@ -97,17 +103,37 @@ def forwardModelPair(
         only specified for the intra or extrafocal donut, both donuts use
         that angle. If neither is specified, the same random angle is used
         for both. (the default is None)
+    miscenterIntra : tuple, optional
```
Comment (jbkalmbach) [inline python/lsst/ts/wep/utils/modelUtils.py:106]: Why is it useful to have the default not be 0? Is it worth the bit of confusion that it may cause when somebody may assume that not entering anything in this optional parameter will still create a non-zero value?

File: python/lsst/ts/wep/utils/modelUtils.py [hunk 2]
```diff
@@ -178,25 +204,38 @@ def forwardModelPair(
...
     if seeing > 0:
         intraStamp.image = convolve(intraStamp.image, atmKernel, mode="same")
-    # Normalize the flux
-    intraStamp.image *= fluxIntra * (1 + nBlends) / intraStamp.image.sum()
-    # Poisson noise
+    # Pseudo-Poissonian noise
     intraStamp.image += background
     intraStamp.image = np.clip(intraStamp.image, 0, None)
-    intraStamp.image = rng.poisson(intraStamp.image).astype(float)
+    intraStamp.image += rng.normal(size=intraStamp.image.shape) * np.sqrt(
```
Comment (jbkalmbach) [inline python/lsst/ts/wep/utils/modelUtils.py:239]: Just trying to understand why this change?

**Review by jbkalmbach — APPROVED** [review]
Looks good. Minor comments.

File: python/lsst/ts/wep/utils/modelUtils.py (hunk 2 above)
Comment (jfcrenshaw) [reply python/lsst/ts/wep/utils/modelUtils.py:239]: With the Poisson, numpy sometimes complains "lam value too large". It's supposedly an issue with the float type in the array. I tried casting to different float types but didn't get it working. I decided it wasn't worth spending more time on, so just changed it to pseudo-Poissonian, which doesn't hit that error ¯\_(ツ)_/¯

File: python/lsst/ts/wep/utils/modelUtils.py (hunk 1 above)
Comment (jfcrenshaw) [reply python/lsst/ts/wep/utils/modelUtils.py:106]: I added it because using perfectly centered donuts causes some asymmetry in errors for Zernikes that are azimuthal pairs. I assume it's just a pixel aliasing thing. Adding small, random dithers fixes that issue. This amount of miscentering doesn't cause problems for the TIE or Danish, and I don't think it's bad to assume that stamps aren't perfectly centered. I'd be fine to set the default to 0 though if you think that's preferable.

Comment (jbkalmbach) [reply python/lsst/ts/wep/utils/modelUtils.py:106]: Since the end product is more robust I think it's good to leave it. Maybe add a version of this explanation somewhere in the docstring or code so we understand when we look at this later.
